### DISCUSSION TITLE
Artifacts size is incorrect?

### DISCUSSION THREAD

**Discussion by LiorBanai** [community/community#25934] [Actions]
<p>Hi,<br>
I’m using the Core 3.1 to build my application with artifacts.<br>
It appears the reported size is incorrect.<br>
for example:<br>
</p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/Analogy-LogViewer/Analogy.LogViewer/runs/909411678" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/431;"><img src="https://user-images.githubusercontent.com/36262995/181094193-f4e749b3-e7f0-481f-aa6c-006c6a8f9543.png" class="thumbnail" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/27751f5e2194bc3d792762edc07eb252adb99d2d_2_690x431.png, https://aws1.discourse-cdn.com/github/original/2X/2/27751f5e2194bc3d792762edc07eb252adb99d2d.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/27751f5e2194bc3d792762edc07eb252adb99d2d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/27751f5e2194bc3d792762edc07eb252adb99d2d_2_10x10.png"></div>

<h3><a href="https://github.com/Analogy-LogViewer/Analogy.LogViewer/runs/909411678" target="_blank" rel="nofollow noopener">Analogy-LogViewer/Analogy.LogViewer</a></h3>

<p>A customizable Log Viewer for most common logs with ability to create custom providers - Analogy-LogViewer/Analogy.LogViewer</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>in this run each zip file is reported as 513 mb but when downloading the file the real size is 215 mb.</p>
<p>am I missing something?</p>

**Comment by Shchvova** [comment, marked as answer]
<p>I seen somewhere that it is done intentionally, showing size of unzipped artifact. Like, automatic top level archiving is ignored.</p>
<p><a href="https://docs.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts" class="onebox" target="_blank" rel="nofollow noopener">https://docs.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts</a></p>
<blockquote>
<p>When an artifact is downloaded using the GitHub UI, all files that were individually uploaded as part of the artifact get zipped together into a single file. This means that billing is calculated based on the size of the uploaded artifact and not the size of the zip file.</p>
</blockquote>

**Comment by LiorBanai** [comment]
<p>oh I see. Thanks! It is kind of confusing.</p>
